### PR TITLE
Add static pages for deploy success and public app instance

### PR DESCRIPTION
**Pull request by tjwp**
Examples of the pages can be seen at 

- https://floating-shelf-13759.herokuapp.com/
- https://floating-shelf-13759.herokuapp.com/success

Prime: @jturkel

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  </p>
+
+  <h2>Description</h2>
+  <p>
+    This service implements the API of the <a href="http://docs.confluent.io/3.0.0/schema-registry/docs/api.html" target="_blank">Confluent Schema Registry</a>.
+    <br/>
+    Imitation is the sincerest form of flattery, right?
+  </p>
+
+  <p>This service is hosted by <a href="http://www.salsify.com">Salsify</a>.</p>
```
Comment (jturkel) [inline public/index.html]: Should we include any kind of text on how you host/manage your own version of this service?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    #container {
+      padding:10px;
+    }
+    .content {
+      padding-top:0em;
+    }
+  </style>
+</head>
+<body>
```
Comment (jturkel) [inline public/index.html]: Can you catch up with @jredburn on any additional branding we want to provide here (e.g. a Salsify logo)?

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class PagesController < ApplicationController
+
+  layout false
+
+  # This page is primarily for the benefit of the schema registry hosted
+  # at avro-schema-registry.salsify.com
+  def index
+    render file: "#{Rails.root}/public/index"
```
Comment (jturkel) [inline app/controllers/pages_controller.rb]: Missing parens?

**Comment by jturkel** [comment]
A few minor comments but looks great. :shipit:

File: public/index.html (hunk 2 above)
Comment (tjwp) [inline public/index.html]: Once this is approved I was also going to ask @emsaka to take a look.

File: app/controllers/pages_controller.rb (hunk 3 above)
Comment (tjwp) [inline app/controllers/pages_controller.rb]: Added

File: public/index.html (hunk 1 above)
Comment (tjwp) [inline public/index.html]: Good call. I'm thinking along the lines of "Host your own copy of the service on Heroku:" and include the Heroku button.

File: public/index.html (hunk 2 above)
Comment (tjwp) [inline public/index.html]: Discussed offline -- keeping it subtle; no logo.

**Pull request merged**